packages/core/src/DreifussWysiwygEditorDemo.tsx

```diff
@@ -177,7 +177,13 @@ const value: any = [
 const DreifussWysiwygEditorDemo = () => (
   <div style={{overflowX: 'hidden', minHeight: 400}}>
     <h1>RichText Component Demo</h1>
-    <DreifussWysiwygEditor showCharCount initialValue={value} />
+    <DreifussWysiwygEditor
+      charactersCount={count => {
+        // console.log(count)
+      }}
+      showCharCount
```

Comment:
Rename this to `showCharactersCount` and set it to `true`. Always work with values.